content/authentication/keeping-your-account-and-data-secure/sudo-mode.md

```diff
@@ -22,7 +22,7 @@ To maintain the security of your account when you perform a potentially sensitiv
 - Authorization of a third-party application
 - Addition of a new SSH key
 
-After you authenticate to perform a sensitive action, your session is temporarily in "sudo mode." In sudo mode, you can perform sensitive actions without authentication. {% data variables.product.product_name %} will wait a few hours before prompting you for authentication again. During this time, any sensitive action that you perform will reset the timer.
+After you authenticate to perform a sensitive action, your session is temporarily in "sudo mode." In sudo mode, you can perform sensitive actions without authentication. {% data variables.product.product_name %} {% ifversion ghes %}has a two-hour session timeout period{% else %}will wait a few hours{% endif %} before prompting you for authentication again. During this time, any sensitive action that you perform will reset the timer.
```

Comment:
```suggestion
After you authenticate to perform a sensitive action, your session is temporarily in "sudo mode." In sudo mode, you can perform sensitive actions without authentication. {% data variables.product.product_name %} has a two-hour session timeout period before prompting you for authentication again. During this time, any sensitive action that you perform will reset the timer.
```

We don't need to hide this detail on dotcom when it's pretty obvious 
```suggestion
After you authenticate to perform a sensitive action, your session is temporarily in "sudo mode." In sudo mode, you can perform sensitive actions without authentication. {% data variables.product.product_name %} has a two-hour session timeout period before prompting you for authentication again. During this time, any sensitive action that you perform will reset the timer.
```

This is true for all versions, so I don't see much reason to obscure this given it can be experimentally determined. The only reason to leave this vague is so we can change it without updating the docs - I'm in favor of taking on that extra burden over leaving this vague for that sake.